backend/sse/src/server.ts

```
@@ -55,9 +79,17 @@
     });
 }
 
-// Create a gRPC server
+const streamReferenceSolution = async (call: any, callback: Function) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)